Specs/DateTime/German/MergedExtractor.json

```diff
@@ -1 +1,144 @@
-[]
\ No newline at end of file
+[{
+    "Input": "das sind 2 Tage",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "2 Tage",
+        "Type": "duration"
+      }
+    ]
+  }
+  ,
+  {
+    "Input": "es ist 4 Uhr",
+    "NotSupportedByDesign": "python",
+    "NotSupported":"javascript",
+    "Results": [
+      {
+        "Text": "4 Uhr",
+        "Type": "time"
+      }
+    ]
+  }
+  ,
+  {//should be datetime
```

Comment:
What do you mean? It should be recognized as datetime, but due to a bug falls here?

Reply:
The set-extractor seems to be a little bit to "dominant",  looking further into this issue today, forgot to put the testcases back to not supported